tools/src/vendoring/config/expoGoConfig.ts

```diff
@@ -167,38 +167,38 @@ const config: VendoringTargetConfig = {
               find: /\b(task (prepareHermes|unpackReactNativeAAR).*\{)$/gm,
               replaceWith: `$1\n    return`,
             },
-            {
-              // remove jsc extraction
-              paths: 'build.gradle',
-              find: /def jscAAR = .*\n.*extractSO.*jscAAR.*$/gm,
-              replaceWith: '',
-            },
+            // {
+            //   // remove jsc extraction
+            //   paths: 'build.gradle',
+            //   find: /def jscAAR = .*\n.*extractSO.*jscAAR.*$/gm,
+            //   replaceWith: '',
+            // },
             {
               // compileOnly hermes-engine
               paths: 'build.gradle',
               find: /implementation "com\.facebook\.react:hermes-android:?"\s*\/\/ version substituted by RNGP/g,
               replaceWith:
                 'compileOnly "com.facebook.react:hermes-android:${REACT_NATIVE_VERSION}"',
             },
-            {
-              // find rn libs in ReactAndroid build output
-              paths: 'CMakeLists.txt',
-              find: 'set (RN_SO_DIR ${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/first-party/react/jni)',
-              replaceWith:
-                'set (RN_SO_DIR "${REACT_NATIVE_DIR}/ReactAndroid/build/intermediates/library_*/*/jni")',
-            },
-            {
-              // find hermes from prefab
-              paths: 'CMakeLists.txt',
-              find: /(string\(APPEND CMAKE_CXX_FLAGS " -DJS_RUNTIME_HERMES=1"\))/g,
-              replaceWith: `find_package(hermes-engine REQUIRED CONFIG)\n    $1`,
-            },
-            {
-              // find hermes from prefab
-              paths: 'CMakeLists.txt',
-              find: /"\$\{BUILD_DIR\}\/.+\/libhermes\.so"/g,
-              replaceWith: `hermes-engine::libhermes`,
-            },
+            // {
+            //   // find rn libs in ReactAndroid build output
+            //   paths: 'CMakeLists.txt',
+            //   find: 'set (RN_SO_DIR ${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/first-party/react/jni)',
+            //   replaceWith:
+            //     'set (RN_SO_DIR "${REACT_NATIVE_DIR}/ReactAndroid/build/intermediates/library_*/*/jni")',
+            // },
+            // {
+            //   // find hermes from prefab
+            //   paths: 'CMakeLists.txt',
+            //   find: /(string\(APPEND CMAKE_CXX_FLAGS " -DJS_RUNTIME_HERMES=1"\))/g,
+            //   replaceWith: `find_package(hermes-engine REQUIRED CONFIG)\n    $1`,
+            // },
+            // {
+            //   // find hermes from prefab
+            //   paths: 'CMakeLists.txt',
+            //   find: /"\$\{BUILD_DIR\}\/.+\/libhermes\.so"/g,
+            //   replaceWith: `hermes-engine::libhermes`,
+            // },
```

Comment:
thanks for commenting out these. i think these code are not necessary now. let's just remove them.